image_encode.py

```diff
@@ -0,0 +1,30 @@
+"""
+For encoding images returned by the camera device.
+"""
+import cv2
+import numpy as np
+
+def encode_image_as_bytes(ext: str, image: "np.ndarray") -> "tuple[bool, bytes | None]" :
+    """
+    Encodes an image (np.ndarray as an RGB matrix) and returns its byte sequence.
+
+    Parameters
+    ----------
+    ext: Desired image encoding format (e.g., ".png").
```

Comment:
Remove this since the encoded format is constant (there won't be a random switch at runtime).